smac/facade/smac_bohb_facade.py

```diff
@@ -12,7 +10,7 @@
 __license__ = "3-clause BSD"
 
 
-class BOHB4HPO(SMAC4AC):
+class BOHB4HPO(SMAC4HPO):
```

Comment:
Shouldn't we also for the sake of clarity explicitly set the Hyperband parameters, e.g. eta, here as well?